packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -103,7 +103,8 @@ function makeAugmentedSchema(
         userCustomResolvers?: IResolvers | Array<IResolvers>;
         subgraph?: Subgraph;
     } = {},
-    schemaModel: Neo4jGraphQLSchemaModel
+    schemaModel: Neo4jGraphQLSchemaModel,
+    _experimentalSchema: boolean
```

Comment:
What is the reason for this experimental flag not being part of the object passed to `makeAugmentedSchema`?

Reply:
it felt like those options are kind of user-driven options if that makes sense, and this one is externally controlled by us so in my view it was polluting that options object. However I don't feel strongly about this

Reply:
fair enough